autogpt/api_manager.py

```diff
@@ -65,32 +65,6 @@ def create_chat_completion(
         self.update_cost(prompt_tokens, completion_tokens, model)
         return response
 
-    def embedding_create(
```

Comment:
This class has too much scope and insufficiently encapsulates existing code.  Next PR will leave it well defined around resource management only.